Packs/GitHub/Integrations/GitHub/GitHub.yml

```diff
@@ -2950,6 +2950,53 @@ script:
     description: Commits a given file.
     execution: false
     name: Github-commit-file
+  - arguments:
+    - default: false
+      description: The name of the release.
+      isArray: false
+      name: name
+      required: false
+      secret: false
+    - default: false
+      description: The name of the releae tag.
+      isArray: false
+      name: tag_name
+      required: true
+      secret: false
+    - default: false
+      description: Text describing the contents of the tag.
+      isArray: false
+      name: body
+      required: false
+      secret: false
+    - default: false
+      description: True to create a draft (unpublished) release, false to create a published one.
+      isArray: false
+      name: draft
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'True'
+      - 'False'
+      defaultValue: 'True'
+    deprecated: false
+    description: Create a release.
+    execution: false
+    name: GitHub-create-release
```

Comment:
once a release is created as a draft, is there an option to publish it?
if not, add it in the command's description as a note

Reply:
Its possible to publish the release after it created as draft with [update release request](https://docs.github.com/en/rest/reference/repos#update-a-release) do you think we need to add it as other command?